webapp/components/webrtc/components/connecting_screen.jsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Actually we can modify loading_screen.jsx to has a message prop and remove this component